.github/workflows/label.yml

```diff
@@ -1,30 +1,36 @@
 name: "Pull Request Labeler"
 on:
-  pull_request_target:
-    types: [opened, reopened]
+  pull_request:
```

Comment:
When using `pull_request_target`, GitHub complained about the syntax of the YML file.As soon as I changed this to `pull_request`, the error dissapeared. I am not sure why this happens...

Even if this workflow is executed every time a push is made, it will not clutter the current conversation in the PR.

Reply:
That's interesting. You might want to open an issue in the corresponding repo: 

https://github.com/actions/labeler

Reply:
Sure! Thanks for the suggestion, @germa89.

Reply:
Linking here the issues I just opened: https://github.com/actions/labeler/issues/360